docs/flow/modin/core/execution/dispatching.rst

```diff
@@ -0,0 +1,54 @@
+:orphan:
+
+..
+    TODO: add links to documentation for mentioned modules.
+
+Factories Module Description
+""""""""""""""""""""""""""""
+
+Brief description
+'''''''''''''''''
+Modin has several execution engines and storage formats, combining them together forms certain backends. 
+Calling any DataFrame API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to
```

Comment:
```suggestion
Calling any Dataframe API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to
```
where needed

Reply:
Am I right that `DataFrame` is for `modin.pandas.DataFrame` and `Dataframe` is for `modin.core.dataframe`? Is so, then this line should be changed since it's describing high-level DataFrame object

Reply:
I prefer we explicit referred to any kind of class. So here we should use ":py:class:`~modin.pandas.DataFrame`".

Reply:
done